data/scripts/lib/register_achievements.lua

```diff
@@ -193,7 +193,7 @@ ACHIEVEMENTS = {
 	[192] = { name = "I Like it Fancy", grade = 1, points = 1, secret = true, description = "You definitely know how to bring out the best in your furniture and decoration pieces. Beautiful." },
 	[193] = { name = "Skin-Deep", grade = 2, points = 4, secret = true, description = "You always carry your obsidian knife with you and won't hesitate to use it. You've skinned countless little - and bigger - critters and yeah: they usually don't get any more beautiful on the inside. It's rather blood and gore and all that..." },
 	[194] = { name = "Ashes to Dust", grade = 2, points = 4, secret = true, description = "Staking vampires and demons has almost turned into your profession. You make sure to gather even the tiniest amount of evil dust particles. Beware of silicosis." },
-	[195] = { name = "Silent Pet", grade = 1, points = 1, secret = true, description = "Awww. Your very own little goldfish friend - he's cute, he's shiny and he can't complain should you forget to feed him. He'll definitely brighten up your day!" },
+	-- [195] = Unknown/non-existent
```

Comment:
I don't know if it's really necessary to maintain this, since we use "pairs" which don't follow an order. If confirmed that it doesn't exist (you can tell by the "protobuf" file in the cipsoft client, then it's best to just remove it.

Reply:
I've been put this, to create an organization, and inform further maintainer that, this achievement doesn't exists.

Reply:
> I don't know if it's really necessary to maintain this, since we use "pairs" which don't follow an order. If confirmed that it doesn't exist (you can tell by the "protobuf" file in the cipsoft client) then it's best to just remove it.

I've been put this, to create an organization, and inform further maintainer that, this achievement doesn't exists.

Reply:
I don't think there's any problem with maintaining it.